sensorhub-driver-android/AndroidManifest.xml

```diff
@@ -2,4 +2,6 @@
     package="org.sensorhub.impl.sensor.android"
     android:versionCode="1"
     android:versionName="1.0" >
+
+    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
I don't think we need this here since we have it on the app manifest. Can you try without?

Reply:
Pushed up a fix for both.